### PR TITLE
Api generated

### PR DESCRIPTION
**Pull request by JKamsker**
@Lewiscowles1986
Idk what i should think of this...
Maybe i should keep the old one and just link to the redoc ?

### CODE REVIEW

**Comment by Lewiscowles1986** [comment]
It's a lot to maintain. Also the deprecated format doesn't say deprecated.

The reason I suggested OpenAPI was to allow the folks at SwaggerHub to generate client / server code was so that you didn't need to own this.

That said, you've made the effort now.

File: README.md [hunk 1]
```diff
@@ -1,51 +1,471 @@
...
-- Validate the responses. 
-	- Re-Calculate the hash of the response ( [Sample HashAlgorithm](https://github.com/audible-tools/audible-tools.github.io/blob/926a3b6ea8f3bc0d71f294adc78d4bbd545c20ab/src/Utils/AaxHashAlgorithm.js) | [Sample Usage](https://github.com/audible-tools/audible-tools.github.io/blob/926a3b6ea8f3bc0d71f294adc78d4bbd545c20ab/src/ChecksumResolver.js#L150))
-- Cache as much as you can
-	- Even though, the server is very efficient at what it does, there is no reason to stress things
+> Scroll down for code samples, example requests and responses. Select a language for code samples from the tabs above or the mobile navigation menu.
 
+<h1 id="aax-activationservice-api_3-activation">Activation</h1>
 
-## v2
+## Resolves
```
Comment (Lewiscowles1986) [inline README.md:13]: Resolves was for me a confusing heading. What was it meant to convey?

**Comment by JKamsker** [comment]
> It's a lot to maintain. Also the deprecated format doesn't say deprecated.
> 
> The reason I suggested OpenAPI was to allow the folks at SwaggerHub to generate client / server code was so that you didn't need to own this.
> 
> That said, you've made the effort now.

I mean its autogenerated with that https://github.com/Mermade/widdershins tool...no effort
Whats swaggerhub?

File: README.md [hunk 2]
```diff
@@ -1,51 +1,471 @@
...
 
-Request:
+> Code samples
```
Comment (Lewiscowles1986) [inline README.md:15]: Did you know you can wrap code samples in?

```
<details>
<summary>TITLE</summary>

// CODE

</details>
```

It helps hide potentially scary code from people who don't want it, while also satiating coders.

File: README.md [hunk 3]
```diff
@@ -1,51 +1,471 @@
...
+        string responseBody = await response.Content.ReadAsStringAsync();
+
+        //Deserialize Body to object
+        var result = JsonConvert.DeserializeObject(responseBody);
+    }
+}
+
+```
+
+`GET /api/v1/Activation/{checksum}`
```
Comment (Lewiscowles1986) [inline README.md:103]: I'd try to find a way to mark this as deprecated, and move it lower in the document. Perhaps under a heading for deprecated endpoints.

**Comment by Lewiscowles1986** [comment]
I Pointed you at https://editor.swagger.io yesterday. It's a commercial company that provides tools for people to work with OpenAPI yaml documents.

**Comment by JKamsker** [comment]
Ah ok sorry, i missed that one i suppose ... and how do am i supposed to share the api on github?

Should we add a link on the main repo?

File: README.md (hunk 2 above)
Comment (JKamsker) [reply README.md:15]: Omg thats awesome

**Comment by Lewiscowles1986** [comment]
So with the yaml file linked in one commit, you can then link to the raw file in another 👍 the editor supports opening URL's.

image

**Comment by JKamsker** [comment]
@Lewiscowles1986 But i'd like to have a permalink  like https://editor.swagger.io?api=https://aax.api.j-kit.me/swagger/v1/swagger.json
I dont like maintaining multiple endpoints. I'd rather have smth like a single source of truth. In the best case it would be generated by the api server i have to document anyway.

**Comment by Lewiscowles1986** [comment]
It's entirely up to you. Please don't take any enthusiasm to be an attempt to give orders.

What you as the owner of the service feel comfortable with, should be what you go with.

**Comment by JKamsker** [comment]
> It's entirely up to you. Please don't take any enthusiasm to be an attempt to give orders.
> 
> What you as the owner of the service feel comfortable with, should be what you go with. Until enough users support or challenge it 😉 .

Haha yea ik, but hearing other opinions is also woth alot.

Ive changed https://github.com/audible-tools/Api-Documentation a bit...
Added ReDoc, Postman and the OpenApi json directlink. Do you think that would be sufficient? If yes, im gonna close this pr...
Do you see anything odd there?

**Pull request closed without merging**